sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala

```diff
@@ -127,10 +128,16 @@ class QueryExecution(
     ReuseExchange(sparkSession.sessionState.conf),
     ReuseSubquery(sparkSession.sessionState.conf))
 
-  def simpleString: String = withRedaction {
+  def simpleString: String = simpleString(false)
+
+  def simpleString(formatted: Boolean): String = withRedaction {
```

Comment:
do we still need the changes in this file?

Reply:
@cloud-fan Yeah.. actually there are quite a few callers of simpleString from test suites and other places like Dataset.explain(). They invoke the function like : 
```
queryExecution.simpleString
```
so i wanted to avoid changing all the callers..